### PR TITLE
Add Quick.JS to jsdelivr

### PR DESCRIPTION
**Pull request by MK2018**
This is a pull request with both the info.ini & the update.json, as per the contribution guidlines, for adding Quick.JS to jsdelivr

### CODE REVIEW

File: files/quickjs/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "quickjs",
+  "repo": "mk2018/quickjs",
+  "files": {
+    "basePath": "builds/0.1a/",
```
Comment (jimaek) [inline files/quickjs/update.json]: This format wont work, it will break the minute you release a new version.
Also there is no documentation or website.

**Comment by MK2018** [comment]
I changed the file structure so that it wouldn't break, and added some basic documentation. As of right now, Quick.JS is still in development, so more documentation is forthcoming, but it'll be done very soon. Hopefully this helps.

**Comment by jimaek** [comment]
I recommend waiting until the code is more stable and production ready before submitting.
Leave a comment here to review it again.

**Comment by MK2018** [comment]
Thanks, I'll do a new pull request when it's ready.

**Pull request closed without merging**